### PR TITLE
[944] Set up different DFE namespace for Find

### PR DESCRIPTION
**Pull request by darokel**
### Context

With Find being merged into Publish, we're going to stream events into one table but setup a different namespace for Find. 

https://trello.com/c/6nkOr3CU/944-handle-dfe-analytics-configuration

### Changes proposed in this pull request

- Sets up a namespace for Find
- Tweaks the DFE config to send jobs async via diff. queue 

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by darokel** [comment]
[Handle DFE analytics configuration](https://trello.com/c/6nkOr3CU/944-handle-dfe-analytics-configuration)

File: app/controllers/find/application_controller.rb [hunk 1]
```diff
@@ -10,6 +12,16 @@ def render_feedback_component
       @render_feedback_component = true
     end
 
+    # For DfE::Analytics as Find doesn't have a current_user
+    def current_user
```
Comment (duncanjbrown) [inline app/controllers/find/application_controller.rb:16]: You can leave this out as `dfe-analytics` will check whether the method exists before calling it:

https://github.com/DFE-Digital/dfe-analytics/blob/b916b441d1a5579465c52d0772d99dab8e072d73/lib/dfe/analytics/requests.rb#L21

**Review by duncanjbrown — COMMENTED** [review]
Looking good! Do you need to include `DfE::Analytics::Requests` in the Publish and Support namespaces too?

**Comment by darokel** [comment]
> Looking good! Do you need to include `DfE::Analytics::Requests` in the Publish and Support namespaces too?

This is already included in the `ApplicationController` specific to publish/support which they both inherit from https://github.com/DFE-Digital/publish-teacher-training/blob/a30b6a691cf7061a738859f0b0734f860323331b/app/controllers/application_controller.rb#L2

**Review by duncanjbrown — APPROVED** [review]
LGTM!

**Pull request merged**